helix-view/src/gutter.rs

```diff
@@ -181,20 +189,26 @@ pub fn line_numbers<'doc>(
                 linenr
             };
 
+            let width = max(n_last_line, n_min);
             write!(out, "{:>1$}", display_num, width).unwrap();
             Some(style)
         }
     })
 }
 
-pub fn line_numbers_width(_view: &View, doc: &Document) -> usize {
+/// The width of a "line-numbers" gutter
+///
+/// The width of the gutter depends on the number of lines in the document,
+/// whether there is content on the last line (the `~` line), and the
+/// `editor.gutters.line-numbers.min-width` settings.
+pub fn line_numbers_width(view: &View, doc: &Document) -> usize {
```

Comment:
It looks like this might be unused. Could it be used above? Also we can probably drop the `pub` if it won't be used elsewhere

Reply:
> It looks like this might be unused.

This is used above in `GutterType::width()`

https://github.com/helix-editor/helix/blob/097fd4f32298b5327490691dea3e8b880890d84b/helix-view/src/gutter.rs#L38-L45

> Could it be used above?

I'm not sure I follow. Would you prefer that it's inlined into `GutterType::width()`?

Reply:
Ah I see, I missed that reference.

I think that the `line_numbers_width` function could be reused in the `line_numbers` above it as well though, right?

Reply:
Yeah, that would work. At one point more of the intermediate values were needed in `line_numbers`, but I think `draw_last` is the only thing that is needed in both places.

I think it's worth the minor redundancy to keep the majority of the logic in one place.

I'll make the change so that we reuse `line_numbers_width`